File: Aula4/ComputationalThinkingAula4/src/exercicio3.java
import java.util.Scanner;

public class exercicio3 {

	public static void main(String[] args) {
		//Entrada: O usuário informará 4 valores (n1, n2, n3, n4)
		//Processamento: Multiplicar o primeiro valor pelo terceiro valor e somar o segundo valor e o quarto (n1*n3 = x , n2+n4 = y)
		//Saída: Mostrar o produto (x) do primeiro valor com o terceiro e a soma (y) do segundo com o quarto
		
			int n1, n2, n3, n4, multi, soma;
		
			Scanner leitor = new Scanner (System.in);
		
				System.out.println("Digite quatro números inteiros");
				n1 = leitor.nextInt();
				n2 = leitor.nextInt();
				n3 = leitor.nextInt();
				n4 = leitor.nextInt();
				
				leitor.close();
				
				multi = n1*n3;
				soma =  n2+n4;
				
				System.out.println("O resultado do produto do 1° e 3° números é " +multi);

				System.out.println("O resultado da soma do 2° e 4° números é "  +soma);
		
	}

}

File: aula5e6/exerciciosPython/exercicio3.py
''' O usuário informa quatro valores. O
programa mostra o resultado da multiplicação do primeiro pelo terceiro e
o resultado da soma do segundo com o quarto.'''

n1 = int(input("Digite o primeiro valor: "))
n2 = int(input("Digite o segundo valor: "))
n3 = int(input("Digite o terceiro valor: "))
n4 = int(input("Digite o quarto valor: "))

multiplicacao = n1*n3
soma = n2+n4

print("O resultado da multiplicação do primeiro valor com o terceiro é {} \ne da soma do segundo com o quarto é {}".format(multiplicacao, soma)) #impressão do resultado
#print("o quadrado do número", numero, "é:", quadrado)